src/app/_components/CheckBox/CheckBox.js

```diff
@@ -0,0 +1,172 @@
+'use client';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from '@/utils/cn';
+import { checkBoxVariants } from './styles';
+
+export default function CheckBox({
```

Comment:
🟠 please use named exports

Reply:
changed this to a named export in the last commit

Reply:
Okay, fixed it for real this time